dev/com.ibm.ws.jbatch.cdi.2.1/bnd.bnd

```diff
@@ -0,0 +1,57 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+bVersion=2.1
+
+Bundle-Name: Batch CDI
+Bundle-SymbolicName: com.ibm.ws.jbatch.cdi.2.1.jakarta
+Bundle-Description: Batch support for CDI; version=${bVersion}; Jakarta Enabled
```

Comment:
```suggestion
Bundle-Description: Batch 2.1 support for CDI; version=${bVersion}; Jakarta Enabled
```
Not sure if we really need the `.jakarta` extension on the bundle name.  Also `; Jakarta Enabled` probably isn't necessary either.

In general the bundle and project name being com.ibm.ws.jbatch.2.1.cdi or io.openliberty.jbatch.2.1.cdi makes more sense because it is 2.1 version of jbatch, not of cdi.